IBPSA/Airflow/Multizone/BaseClasses/WindpressureProfile.mo

```diff
@@ -0,0 +1,63 @@
+within IBPSA.Airflow.Multizone.BaseClasses;
+function WindpressureProfile
```

Comment:
I think that the according to the naming conventions we'd use the name `WindPressureProfile` instead of `WindpressureProfile`.

Reply:
As it is function it should start without capital, changed it to windPressureProfile

Reply:
correct